### PR TITLE
Faster XML parsing

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

File: map/style_tests/style_symbols_consistency_test.cpp [hunk 1]
```diff
@@ -14,37 +14,38 @@
...
   {}
 
-  bool Push(string const &) { return true; }
-  void Pop(string const &) {}
-  void CharData(string const &) {}
-  void AddAttr(string const & attribute, string const & value)
+  bool Push(char const *) { return true; }
+  void Pop(char const *) {}
+  void CharData(std::string const &) {}
+  void AddAttr(char const * attribute, char const * value)
```
Comment (vng) [inline map/style_tests/style_symbols_consistency_test.cpp:36]: ```std::string_view attribute``` for nicer compare?

File: drape_frontend/gui/skin.cpp [hunk 2]
```diff
@@ -90,9 +90,9 @@ class ResolverParser
     else if (attr == "vertical" || attr == "horizontal")
     {
       if (m_element == Element::Anchor)
-        m_resolver.AddAnchor(ParseValueAnchor(value));
+        m_resolver.AddAnchor(ParseValueAnchor(std::string_view{value}));
       else if (m_element == Element::Relative)
-        m_resolver.AddRelative(ParseValueAnchor(value));
+        m_resolver.AddRelative(ParseValueAnchor(std::string_view{value}));
```
Comment (vng) [inline drape_frontend/gui/skin.cpp:95]: nit: Take ```std::string_view value``` in Parse definition.

**Review by vng — APPROVED** [review]

File: map/style_tests/style_symbols_consistency_test.cpp (hunk 1 above)
Comment (biodranik) [reply map/style_tests/style_symbols_consistency_test.cpp:36]: The current approach is marginally faster :)

File: drape_frontend/gui/skin.cpp (hunk 2 above)
Comment (biodranik) [reply drape_frontend/gui/skin.cpp:95]: No, there are two cases above that will work faster without a temporary string_view. Now it is created only when necessary.

Comment (vng) [reply drape_frontend/gui/skin.cpp]: Ok, then write simple ParseValueAnchor(value)

Comment (biodranik) [reply drape_frontend/gui/skin.cpp]: Done.

**Pull request merged**